### DISCUSSION TITLE
MacOs Sandbox

### DISCUSSION THREAD

**Discussion by fufufu3** [AvaloniaUI/Avalonia#7423] [Q&A] [answered]
my app with avalonia,it run fine.but when i added SandBox,it crash every time.
the error in Console 
`Fontconfig error: Cannot load default config file 
xxx: segmentation fault`
the error in MacOs report 
`Crashed Thread:        0  Dispatch queue: com.apple.main-thread

Exception Type:        EXC_BAD_ACCESS (SIGSEGV)
Exception Codes:       KERN_INVALID_ADDRESS at 0x0000000000000068
Exception Note:        EXC_CORPSE_NOTIFY

Termination Signal:    Segmentation fault: 11
Termination Reason:    Namespace SIGNAL, Code 0xb
Terminating Process:   exc handler [1118]

VM Regions Near 0x68:
--> 
    __TEXT                 000000010bc9b000-000000010c4d7000 [ 8432K] r-x/r-x SM=COW  /Users/USER/Desktop/*/xxx.app/Contents/MacOS/xxxx

Thread 0 Crashed:: Dispatch queue: com.apple.main-thread
0   libsystem_c.dylib             	0x00007fff6b9ccba4 flockfile + 18
1   libsystem_c.dylib             	0x00007fff6b9cfc50 fwrite + 66
2   libgdiplus.dylib              	0x000000012029aeee GdiplusStartup + 222
3   ???                           	0x00000001251aa7a7 0 + 4917471143
4   ???                           	0x00000001251a94af 0 + 4917466287
5   ???                           	0x0000000125197241 0 + 4917391937
6   com.xxx.xxxxx            	0x000000010bfda3e9 CallDescrWorkerInternal + 124
7   com.xxx.xxxxx            	0x000000010be24ba6 DispatchCallDebuggerWrapper(CallDescrData*, int) + 230
8   com.xxx.xxxxx             	0x000000010be24df6 DispatchCallSimple(unsigned long*, unsigned int, unsigned long, unsigned int) + 294
9   com.xxx.xxxxx             	0x000000010bda45c3 MethodTable::RunClassInitEx(Object**) + 371
10  com.xxx.xxxxx             	0x000000010bda4aa8 MethodTable::DoRunClassInitThrowing() + 712
11  com.xxx.xxxxx            	0x000000010be8ba37 JIT_GetSharedNonGCStaticBase_Helper + 199`

I can't understand this mistake,If you have a similar problem, may you tell me how did you solve it.
Thank you very much

**Comment by kekekeks** [comment, marked as answer]
Looking at the stacktrace your issue is caused by libgdiplus.dylib, which is an emulation layer for System.Drawing. It looks like that it's not compatible with sandbox

**Reply by fufufu3** [reply]
oh  i solve the error.thanks very much, bro
